### PR TITLE
[ET-918] don't escape the title

### PR DESCRIPTION
**Pull request by cliffordp**
🎫 [ET-918]

no artifact needed

[ET-918]: https://moderntribe.atlassian.net/browse/ET-918

### CODE REVIEW

File: src/views/v2/tickets/item/content/title.php [hunk 1]
```diff
@@ -36,7 +36,7 @@
 <div <?php tribe_classes( $title_classes ); ?> >
 	<?php if ( $is_mini ) : ?>
 		<div <?php tribe_classes( $event_title_classes ); ?> >
-			<?php echo esc_html( get_the_title( $post_id ) ); ?>
+			<?php echo get_the_title( $post_id ); ?>
```
Comment (tr1b0t) [inline src/views/v2/tickets/item/content/title.php:39]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_the_title'.

<sub>hash: 09202ed14cdd0f25c2329d0c646b6161</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

File: src/views/v2/tickets/item/content/title.php (hunk 1 above)
Comment (cliffordp) [reply src/views/v2/tickets/item/content/title.php:39]: @sc0ttkclark ☝️

Comment (juanfra) [reply src/views/v2/tickets/item/content/title.php:39]: I think we should want to go with `wp_kses_post()` here.

**Review by juanfra — COMMENTED** [review]
Nor sure if we defined how we should proceed with title escaping. I feel we would like to use `wp_kses_post()` for titles.

File: src/views/v2/tickets/item/content/title.php (hunk 1 above)
Comment (cliffordp) [reply src/views/v2/tickets/item/content/title.php:39]: @sc0ttkclark said no in the Slack DM I double-checked him with. We'll see; maybe he was tired.

Comment (sc0ttkclark) [reply src/views/v2/tickets/item/content/title.php:39]: Basically the_title() doesn't do anything extra, so there's an intention in regards to titles with their rendering.

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**